unified-codes/apps/data-service/src/app/resolvers.ts

```diff
@@ -61,7 +61,7 @@ export const resolvers = {
 
       // Workaround to prevent interaction requests for multiple entities
       if (info.path.prev?.key == 'entity') {
-        const rxNavIds = parent.has_property?.filter(
```

Comment:
This change results in rxNavIds always being undefined now :( I think we either need to map `has_property` to properties in the entity resolver/schema (probably the proper fix) or keep as `has_property` here. 

I agree with the idea behind the rename though! Maybe better as a separate issue since getting everything merged will be good 😸 .

Reply:
Can fix in a new issue, I think! Easiest is probably to have specific schematic classes/interfaces which share common interfaces, e.g. `IEntity` v. `IEntityType`.

Reply:
Sounds good to me - created #219 for it (and now we've hit 100 comments - this reply was actually kind of redundant but I couldn't resist)

Reply:
![](https://media4.giphy.com/media/3oEjI9byGIPCDolVBK/giphy.gif)